modules/_node-scripts/util/require.mjs

```diff
@@ -0,0 +1,8 @@
+import module from 'module';
+import resolve from 'resolve';
+
+const require = module.createRequire(import.meta.url);
+
+export default function(filePath) {
+	return require(resolve.sync(filePath, {basedir: '.'}));
```

Comment:
Can you not use `import.meta.resolve()`?

Reply:
Not really, because we want to resolve things for the project being built (that's why we use `{basedir: '.'}`) and by using import.meta.resolve or require.resolve you would resolve in the context of `_node-scripts` which is not exactly what we want...

Reply:
We already used the `resolve` package in the bundler and npm-scripts for the very same reason.